### PR TITLE
Tickets/dm 16255

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import distutils
+from distutils.core import setup
+import glob
+
+bin_files = glob.glob("bin/*")
+data_files = [("",["setpath.sh"])]
+
+# The main call
+setup(name='salpytools',
+      version ='OCS_IntSpecICC',
```
Comment (mareuter) [inline setup.py]: Is this really tracking the XML version? I didn't think there was much that was XML specific as this is an abstraction package. How does the version change if changes are only code but not the XML?

Comment (timj) [reply setup.py]: Is this why the header service files are showing me an odd `HEADVER`? I was expecting some kind of number or SHA1 to appear for header version and it should be related to the header service configuration, not the code.

Comment (menanteau) [reply setup.py]: This is a leftover from a previous integration activity. Unfortunately we do not have a method/definition to capture versions.

**Review by mareuter — APPROVED** [review]
Looks good to me with the versions changed.

**Pull request merged**